src/VirtoCommerce.MarketingModule.Core/Model/Promotions/Rewards/PromotionReward.cs

```diff
@@ -37,5 +37,9 @@ protected PromotionReward()
 
         public Promotion Promotion { get; set; }
 
+        /// <summary>
+        /// Promotion reward type. Should be here for proper web model PromotionReward.RewardType filling
+        /// </summary>
+        public string RewardType => GetType().Name;
```

Comment:
I would like to set this property as well as Id explicitly as a parameter passed in a protected ctor. It allows to do not change RewardType when type overriding